CONTRIBUTING.md

```diff
@@ -23,6 +28,49 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Instruction notebook content
```

Comment:
Can we add something about chunk naming in this section? Also would include `live = TRUE` here, which might be somewhere else in the guidelines but I had trouble finding it which suggests it should go where I would expect it to go.

Reply:
Yup! I do not think we have it elsewhere, and this is the logical place.